cli/src/commands/create-stub.js

```diff
@@ -8,7 +8,7 @@ import {findFlowRoot} from '../lib/flowProjectUtils.js';
 import {getPackageJsonData, loadPnpResolver} from '../lib/npm/npmProjectUtils';
 import {path} from '../lib/node';
 
-export function setup(yargs: Object) {
+export function setup(yargs: Object): any {
```

Comment:
If we could also kill those `Object`, would be nice.

Reply:
I'll look into it